tests/rados/pool_tests.py

```diff
@@ -11,7 +11,10 @@
 
 from ceph.ceph_admin import CephAdmin
 from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.pool_workflows import PoolFunctions
 from tests.rados.monitor_configurations import MonConfigMethods
+from tests.rados.stretch_cluster import wait_for_clean_pg_sets
+from utility.utils import method_should_succeed
 
 log = logging.getLogger(__name__)
```

Comment:
let us use custom log class in this file as well.

Reply:
Changed to use the new log class